src/CMakeLists.txt

```diff
@@ -798,6 +809,10 @@ if(WITH_LIBCEPHFS)
     $<TARGET_OBJECTS:common_mountcephfs_objs>)
   target_link_libraries(mount.ceph keyutils)
 
+	if(${WITH_LIBZBC})
+		target_link_libraries(cephfs zbc)
```

Comment:
are we using libzbc in cephfs now? if not, could you postpone this commit until we are?